src/NHSD.BuyingCatalogue.Ordering.Api/Infrastructure/LogHelper.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using Microsoft.AspNetCore.Http;
+using Serilog;
+using Serilog.Events;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Infrastructure
+{
+    public  static class LogHelper
```

Comment:
The naming of this class isn't great – see the [DAPI repo](https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueDocumentService) for an improved implementation.

Reply:
I forgot to add DAPI doesn't include `EnrichFromRequest` but you can probably ignore that omission in this case.

Reply:
DAPI is a much cleaner implementation

Reply:
Updated using DAPI implementation